Nodejs/Product/Nodejs/Workspace/PackageJsonScannerFactory.cs

```diff
@@ -43,13 +43,16 @@ protected override Task<List<FileReferenceInfo>> ComputeFileReferencesAsync(stri
 
                 var packageJson = PackageJsonFactory.Create(filePath);
                 var main = packageJson.Main;
-                var fileReferences = new List<FileReferenceInfo>
+
+                var fileReferences = new List<FileReferenceInfo>();
```

Comment:
>new List<FileReferenceInfo>(); [](start = 37, length = 30)

Seems like it might be (slightly) preferable to return `Array.Empty` if there's no `main`.